package.json

```diff
@@ -9,36 +9,28 @@
   "scripts": {
     "test": "jest --coverage",
     "snyk": "snyk test",
-    "clean":
-      "rimraf ./dist && mkdirp ./dist ./dist/js ./dist/css ./dist/fonts ./dist/img",
-    "js": "rimraf ./dist/js/app.min.js.bak && webpack && npm run env",
-    "renew": "webpack --config webpack.renew.js",
-    "sass":
-      "node-sass --quiet --output-style=compressed --source-map=./dist/css/ ./src/sass/app.scss ./dist/css/app.min.css && cp node_modules/uswds/src/fonts/sourcesanspro-* ./dist/fonts",
-    "html":
-      "html-minifier --collapse-whitespace ./src/index.html -o ./dist/index.html && html-minifier --collapse-whitespace ./src/silent_renew.html -o ./dist/silent_renew.html",
-    "img":
-      "cp ./src/img/min/* ./dist/img && cp -r ./node_modules/uswds/dist/img/* ./dist/img",
-    "env": "./env.sh",
-    "postinstall":
-      "npm run clean && npm rebuild node-sass && npm run build && npm run renew",
-    "watch:js":
-      "nodemon --exitcrash -e js,jsx -w src/js -i src/js/utils/silent_renew.js -x 'npm run js'",
-    "watch:renew":
-      "nodemon --exitcrash -w src/js/silent_renew.js -x 'npm run renew'",
-    "watch:tests":
-      "nodemon --exitcrash -e js -w __tests__ -x 'jest --coverage -o'",
-    "watch:sass": "nodemon --exitcrash -e scss -w src/sass -x 'npm run sass'",
-    "watch:html": "nodemon --exitcrash -e html -w src -x 'npm run html'",
-    "watch":
-      "npm run watch:js & npm run watch:renew & npm run watch:tests & npm run watch:sass & npm run watch:html",
-    "build": "npm run js && npm run sass && npm run html && npm run img"
+    "clean": "rimraf ./dist && mkdirp ./dist ./dist/js ./dist/css ./dist/fonts ./dist/img",
+    "clearBackup": "rimraf ./dist/js/app.min.js.bak",
+    "js": "webpack --progress",
+    "js:dev": "webpack --config webpack.dev.js --progress",
+    "renew": "webpack --config webpack.renew.js --progress",
+    "sass": "node-sass --quiet --output-style=compressed --source-map=./dist/css/ ./src/sass/app.scss ./dist/css/app.min.css && cp node_modules/uswds/src/fonts/sourcesanspro-* ./dist/fonts",
+    "html": "html-minifier --collapse-whitespace ./src/index.html -o ./dist/index.html && html-minifier --collapse-whitespace ./src/silent_renew.html -o ./dist/silent_renew.html",
+    "img": "cp ./src/img/min/* ./dist/img && cp -r ./node_modules/uswds/dist/img/* ./dist/img",
+    "postinstall": "yarn clean && yarn build && yarn renew",
+    "watch:js": "yarn js:dev -- --watch",
+    "watch:sass": "node-sass --watch --quiet --output-style=compressed --source-map=./dist/css/ ./src/sass/app.scss ./dist/css/app.min.css",
+    "watch:tests": "jest --watchAll",
```

Comment:
What do you think about using [`--watch`](https://facebook.github.io/jest/docs/en/cli.html#watch) instead? To get the full report we still have the `test` script.

Reply:
Just --watch is fine, I found it to not always be 100% accurate, but it's probably good enough in most situations. Also, if other tests fail that it doesn't find, that's probably an issue with our test architecture/tests being too coupled.

Reply:
To be honest I'm fine either way. I usually just use `jest --watch` anyway. :) But this is still convenient to have here.